utils/build-script

```diff
@@ -934,6 +950,22 @@ the number of parallel build jobs to use""",
     if args.extra_swift_args:
         build_script_impl_args += ["--extra-swift-args", ";".join(args.extra_swift_args)]
 
+    if args.cross_compile_targets:
```

Comment:
If this option only cross-compiles the standard library, please rename it to make it clear.  As currently named, it makes it sound like it cross-compiles the whole compiler.